app/femr/business/services/system/InternetStatusService.java

```diff
@@ -0,0 +1,60 @@
+package femr.business.services.system;
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import femr.business.services.core.IInternetStatusService;
+import femr.common.IItemModelMapper;
+import femr.common.dtos.ServiceResponse;
+import femr.common.models.InternetStatusItem;
+import femr.data.daos.core.IInternetStatusRepository;
+import femr.data.models.core.*;
+import play.Logger;
+
+public class InternetStatusService implements IInternetStatusService {
+  private final IInternetStatusRepository internetStatusRepository;
+  private final IItemModelMapper itemModelMapper;
+
+  @Inject
+  public InternetStatusService(IInternetStatusRepository internetStatusRepository,
+                        @Named("identified") IItemModelMapper itemModelMapper) {
+
+    this.internetStatusRepository = internetStatusRepository;
+    this.itemModelMapper = itemModelMapper;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public ServiceResponse<InternetStatusItem>  retrieveInternetStatus() {
+
+    ServiceResponse<InternetStatusItem>  response = new ServiceResponse<>();
+
+    try {
+
```

Comment:
Looks like an unneeded blank line. Not sure if this is the style everywhere. If it is then never mind.

Reply:
The style was based on another retrieve method that existed in another service class. You are correct since it looks a bit awkward but it's hard to know which style to use since it's pretty inconsistent in a lot of the classes.